### PR TITLE
ViewPager fixes

### LINKED ISSUE #69: MyMatches: Improperly reported matches
For account AndrewThomas I have 6 matches listed with dog id 1, a few other non-repeated, and bunch of missing matches despite clicking the "like" button on dogs.

### PR DESCRIPTION
**Pull request by Andrewdt97**
Fixes #69 #65 #64 #63

### CODE REVIEW

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java [hunk 1]
```diff
@@ -90,18 +90,29 @@ public void onResponse(String response) {
                         dogs.add(dogsitem);
                     }
 
-
                     viewPager = (ViewPager) findViewById(R.id.pictureWindow);
                     adapter = new ImageAdapter(activity_adoptPet.this,dogs);
                     viewPager.setAdapter(adapter);
+                    viewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+                        public void onPageScrollStateChanged(int state) {
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java:97]: This function does not do anything; do we need it here?

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java [hunk 2]
```diff
@@ -90,18 +90,29 @@ public void onResponse(String response) {
...
+                            TextView name = findViewById(R.id.matchName);
+                            TextView breed = findViewById(R.id.matchBreed);
+                            TextView gender = findViewById(R.id.matchGender);
 
+                            name.setText(dogs.get(position).name_string);
+                            breed.setText(dogs.get(position).breed_string);
+                            gender.setText(dogs.get(position).gender_string);
+                        }
 
+                        public void onPageSelected(int position) {
```
Comment (carolinecarlson) [inline app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java:110]: This function also does not do anything; do we need it here?

**Review by carolinecarlson — COMMENTED** [review]
I added a couple comments; other than that, it looks good.

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java (hunk 1 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java:97]: Yeah we do actually. It gets called with when the state is changed, we just want nothing to happen.

File: app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java (hunk 2 above)
Comment (Andrewdt97) [reply app/src/main/java/edu/calvin/cs262/teamc/activity_adoptPet.java:110]: See above

**Pull request merged**